docs/faq.md

```diff
@@ -1,5 +1,47 @@
 # Frequently Asked Questions
 
+## 6-Minute Delays in Attaching Volumes
+
+### What causes 6-minute delays in attaching volumes?
+
+After a node is preempted, pods using persistent volumes may experience a 6+ minute restart delay. This occurs when a volume is not cleanly unmounted, as indicated by `Node.Status.VolumesInUse`. The Attach/Detach controller in `kube-controller-manager` waits for 6 minutes before issuing a force detach. This delay in pod restart can lead to potential data corruption due to unclean mounts.
+
+### What behaviors have been observed that contribute to this issue?
+
+- The CSI driver on the node may get terminated before the workload's volumes are unmounted, leading to unmount failures.
+- The operating system might shut down before volume teardown is completed, even with remaining time in the `shutdownGracePeriod`.
+
+### What are the symptoms of this issue?
+
+```
+Warning  FailedAttachVolume      6m51s              attachdetach-controller  Multi-Attach error for volume "pvc-4a86c32c-fbce-11ea-b7d2-0ad358e4e824" Volume is already exclusively attached to one node and can't be attached to another
+```
+
+### What steps can be taken to mitigate this issue?
+
+1. **Graceful Termination:** Ensure instances are gracefully terminated, which is a best practice in Kubernetes. This allows the CSI driver to clean up volumes before the node is terminated utilizing the preStop lifecycle hook.
+
+2. **Configure Kubelet for Graceful Node Shutdown:** For unexcpeted shutdowns, it is highly recommended to configure the Kubelet for graceful node shutdown. Using the standard Amazon Linux 2 (AL2) AMI, you can configure the kubelet for graceful node shutdown with the following user data script:
```

Comment:
Should we add something about future EKS optimized AMIs maybe having this by default? Or shall we wait until those AMIs are released. 

Looks like the PR is on hold: [[WIP] Enable graceful node shutdown for 1.29+ #1544](https://github.com/awslabs/amazon-eks-ami/pull/1544)

Reply:
Given the PR is in-flight lets hold off on stating that here for now